web/src/Setting.js

```diff
@@ -648,6 +648,7 @@ export function getProviderTypeOptions(category) {
         {id: "MiniMax", name: "MiniMax"},
         {id: "Local", name: "Local"},
         {id: "Azure", name: "Azure"},
+        {id: "Cohere", name: "Cohere"},

```

Comment:
Code not added for `category === "Embedding"`?

Reply:
"Embedding" condition branch has existed.